cestzam-core/src/test/java/be/valuya/cestzam/core/LoginMyMinfinTest.java

```diff
@@ -0,0 +1,182 @@
+package be.valuya.cestzam.core;
+
+import be.valuya.cestzam.client.error.CestzamClientError;
+import be.valuya.cestzam.core.auth.CestzamAuthService;
+import be.valuya.cestzam.core.auth.emailcode.EmailCodeSession;
+import be.valuya.cestzam.core.auth.emailcode.EmailCodeSessionStatus;
+import be.valuya.cestzam.core.auth.itsme.ItsmeAuthSession;
+import be.valuya.cestzam.core.auth.itsme.ItsmeAuthSessionStatus;
+import be.valuya.cestzam.core.error.CestzamServerError;
+import be.valuya.cestzam.core.service.myminfin.MyminfinServiceProvider;
+import be.valuya.cestzam.core.service.myminfin.MyminfinSession;
+import be.valuya.cestzam.core.service.myminfin.MyminfinSessionStatus;
+import be.valuya.cestzam.core.session.CestzamSession;
+import be.valuya.cestzam.mock.CdiExtension;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.inject.Inject;
+import java.util.Locale;
+
+@ExtendWith(CdiExtension.class) // Runs the test with CDI-Unit
+public class LoginMyMinfinTest {
+
+    @Inject
+    private MyminfinServiceProvider myminfinServiceProvider;
+    @Inject
+    private CestzamAuthService authService;
+    @Inject
+    @ConfigProperty(name = "be.valuya.cestzam.test.itsme.phoneNumber")
+    private String testItsmePhoneNumber;
+    @Inject
+    @ConfigProperty(name = "be.valuya.cestzam.test.emailcode.username")
+    private String testEmaiLCodeUsername;
+    @Inject
+    @ConfigProperty(name = "be.valuya.cestzam.test.emailcode.password")
+    private String testEmaiLCodePassword;
+    @Inject
+    @ConfigProperty(name = "be.valuya.cestzam.test.emailcode.email")
+    private String testEmaiLCodeEmail;
+
+
+    @Test
+    public void testItsmeAuth() throws CestzamServerError, CestzamClientError, InterruptedException {
+        CestzamSession cestzamSession = myminfinServiceProvider.initSession(Locale.FRANCE);
```

Comment:
untested test case.

each service call would correspond to an http request on the api